index.html

```diff
@@ -57,27 +61,27 @@ <h2>Here's a time line of Dr. Borlaug's life:
           <li><b>2009</b> - dies at the age of 95.</li>
         </ul>
 
-      </div>
-      <div id="tribute-text">
+      </section>
+      <section id="tribute-text">
         <p>"Borlaug's life and achievement are testimony to
           the far-reaching contribution <br>that one man's towering intellect, 
           persistence and scientific vision<br> can make to human peace and progress.<br> <br>
           "-- Indian Prime Minister Manmohan Singh </p>
-      </div>
-     <div id="link">
+      </section>
+     <section id="link">
        <h4>
 
         If you have time, you should read more about this incredible human being on his
         <a id ="tribute-link" href="https://en.wikipedia.org/wiki/Norman_Borlaug" target="_blank">Wikipedia entry.</a>
 
        </h4>
```

Comment:
Replace h4 with h3 to   stick on the best practice of using header tag order

Reply:
- [ ] Same here Kindly ask for it politely. and good work here mentioning the reason.

**Please make another review in the same Pull Request regarding my suggestions using inline comments as you did but make it in one review because I can see that you made every required change in a separate review If you don't know how to do it Kindly go back again and watch the tutorial carefully and it will help**